includes/class-wc-gateway-swedbank-pay-invoice.php

```diff
@@ -210,35 +236,50 @@ public function payment_fields() {
 	 */
 	public function validate_fields() {
 		if ( empty( $_POST['billing_country'] ) ) {
-			wc_add_notice( __( 'Please specify country.', WC_Swedbank_Pay::TEXT_DOMAIN ), 'error' );
+			wc_add_notice( __( 'Please specify country.', 'swedbank-pay-woocommerce-payments' ), 'error' );
 
 			return false;
 		}
 
 		if ( empty( $_POST['billing_postcode'] ) ) {
-			wc_add_notice( __( 'Please specify postcode.', WC_Swedbank_Pay::TEXT_DOMAIN ), 'error' );
+			wc_add_notice( __( 'Please specify postcode.', 'swedbank-pay-woocommerce-payments' ), 'error' );
 
 			return false;
 		}
 
-		if ( ! in_array( mb_strtoupper( $_POST['billing_country'], 'UTF-8' ), [ 'SE', 'NO', 'FI' ] ) ) {
-			wc_add_notice( __( 'This country is not supported by the payment system.', WC_Swedbank_Pay::TEXT_DOMAIN ), 'error' );
+		if ( ! in_array( mb_strtoupper( $_POST['billing_country'], 'UTF-8' ), array( 'SE', 'NO', 'FI' ), true ) ) {
+			wc_add_notice(
+				__( 'This country is not supported by the payment system.', 'swedbank-pay-woocommerce-payments' ),
+				'error'
+			);
 
 			return false;
 		}
 
 		// Validate country phone code
-		if ( in_array( $_POST['billing_country'], [ 'SE', 'NO' ] ) ) {
+		if ( in_array( $_POST['billing_country'], array( 'SE', 'NO' ), true ) ) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Direct use of $_POST Superglobal detected.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-woocommerce-payments/pullRequest?prid=5340032)